### ISSUE TITLE
Screen reader does not announce name, role and state for the combo box list items in windows.

### ISSUE THREAD

**Issue by msftedad** [ant-design/ant-design#56070]
### Reproduction link

[https://ant.design/components/tree-select?locale=en-US](https://ant.design/components/tree-select?locale=en-US)

**User Impact:**
People using screen readers cannot identify or navigate through combo box options because the item names, role and state are not announced. This makes it difficult to select the correct value, reducing accessibility and usability for blind users.

### Steps to reproduce

**Pre-Requisites:**
Enable NVDA

**Repro Steps:**
1. Open the URL: https://codesandbox.io/p/sandbox/clyyk8 in edge in windows.
2. 'CodeSandbox' page will appear.
3. Press tab key to navigate to the combo box and activate it.
4. List items will appear.
5. Verify whether screen reader announces name, role and state for the combo box list items in windows or not.

### What is expected?

Screen reader should announce name, role and state for the combo box list items in windows.

### What is actually happening?

Screen reader does not announce name, role and state for the combo box list items in windows.
Screen Reader announces: 0-1, 0-0

**Refer Attachment:**
Screen reader does not announce name, role and state for the combo box list items in windows.png
Screen reader does not announce name, role and state for the combo box list items in windows.mp4

https://github.com/user-attachments/assets/1c02678d-dfcc-46f4-9a1e-5319e942b2bb
<img width="973" height="532" alt="Image" src="https://github.com/user-attachments/assets/76e99d2d-8689-4ea5-b0e6-38f7b6c69cdd" />

| Environment | Info |
| --- | --- |
| antd | 5.27.6 |
| React | 18 |
| System | Windows 11 Version25H2 (OS Build 26220.7271) |
| Browser | Edge (Version 143.0.3650.46) |

**Comment by msftedad** [comment]
Hi, Update from Eng team: Tried the fix provided in above comment, Disabling Virtual but fix didn't work.

**Comment by msftedad** [comment]
Hi Team, any update on this issue?
